shell/browser/api/electron_api_service_worker_context.cc

```diff
@@ -87,9 +87,10 @@ ServiceWorkerContext::~ServiceWorkerContext() {
 
 void ServiceWorkerContext::OnReportConsoleMessage(
     int64_t version_id,
+    const GURL& scope,
     const content::ConsoleMessage& message) {
   v8::Isolate* isolate = JavascriptEnvironment::GetIsolate();
-  v8::HandleScope scope(isolate);
+  v8::HandleScope handle_scope(isolate);
   Emit("console-message",
```

Comment:
Should we add `scope` to the event object?

Reply:
Seems like we do now https://github.com/electron/electron/blob/master/shell/common/gin_helper/event_emitter.h#L63 , can remove this in a follow-up PR.